packages/expo-clipboard/src/Clipboard.types.ts

```diff
@@ -38,3 +38,32 @@ export interface ClipboardImage {
     height: number;
   };
 }
+
+export enum StringFormat {
+  PLAIN_TEXT = 'plainText',
+  HTML = 'html',
+}
+
+export interface GetStringOptions {
+  /**
+   * The target format of the clipboard string to be converted to, if possible.
+   * Defaults to plain text.
+   *
+   * On web, this option is ignored. The string is always returned without any conversion.
```

Comment:
It's not doable? 🤔

Reply:
It is (in a limited way - there's no built-in support for html->plaintext conversion), but it needs some web progress first (other PRs are coming), I'll try to do it in a separate PR

Reply:
Created a linear task to track this: ENG-4329